client/connector_token_create.go

```diff
@@ -99,67 +99,68 @@ func configureHostPorts(result *RouterHostPorts, cli *VanClient, namespace strin
 	if namespace == "" {
 		namespace = cli.Namespace
 	}
-	ok, err := configureHostPortsFromRoutes(result, cli, namespace)
```

Comment:
I *believe* that the logic for cert tokens already checks whether routes are actually in use. The problem is with claim tokens.

Reply:
Correct, it is not returning an error when not found occurs. I will revert it.